modules/installation-requirements-user-infra.adoc

```diff
@@ -80,6 +102,65 @@ Each cluster machine must meet the following minimum requirements:
 
 |===
 
+ifeval::["{context}" == "installing-ibm-z"]
+[id="minimum-ibm-z-system-requirements_{context}"]
+== Minimum IBM Z system requirements
+
+=== Hardware requirements
+
+* 1 LPAR with 3 IFLs that supports SMT2
+* 1 OSA or RoCE network adapter, or both
+
+=== Operating system requirements
+
+* One instance of z/VM 7.1
+
+On your z/VM instance, set up:
+
+* 3 guest virtual machines for OCP control plane machines (master nodes)
+* 3 guest virtual mchines  for OCP compute machines (worker nodes)
+* 1 guest virtual machine for the temporary OCP bootstrap machine
+
+=== Disk storage for the z/VM guest virtual machines 
```

Comment:
Consider removing the assembly prereq about storage.

Reply:
Discussed with David and Jeremy, decided to leave both sections.